### DISCUSSION TITLE
Pull requests don’t get updated

### DISCUSSION THREAD

**Discussion by NikhilMath** [community/community#27560] [Other Feature Feedback, Questions, & Ideas] [answered]
I do not see my pull request shark ever getting updated, I am pretty sure more than 2 of my pull requests have been approved lol.

**Comment by radar-blip** [comment, marked as answer]
It will only get updated for every tier (level)  2 - 16 - 128 - 1024   so if you get every tier, you will have  x4
Here is a good guide: https://github.com/drknzz/GitHub-Achievements

So, you will get x3 at 128 pull requests.

**Comment by jfullstackdev** [comment]
take a look at mine 

![image](https://user-images.githubusercontent.com/47092464/181703026-da0b5b5a-271f-4ce7-8fc1-2c342162871b.png)

it's 2 , 16 , 128

**Reply by NikhilMath** [reply]
Thank you! So should it be permanently bronze at 128 pull requests?

**Reply by radar-blip** [reply]
At 128, it will turn to silver (x3). And when you hover over the badge, it will flip back and forth between bronze and silver, showing you completed 3 tiers (and 128 pull requests).
